integration/data_source_lacework_user_profile.go

```diff
@@ -0,0 +1,22 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestUserProfileDataSource uses the Terraform plan at:
+// => '../examples/data_source_lacework_user_profile'
+func TestUserProfileDataSource(t *testing.T) {
+	// Read API access token
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../examples/data_source_lacework_user_profile",
+	})
+	defer terraform.Destroy(t, terraformOptions)
+
+	terraform.InitAndApplyAndIdempotent(t, terraformOptions)
+	url := terraform.Output(t, terraformOptions, "lacework_user_profile_url")
+	assert.NotEmpty(t, url)
```

Comment:
nit: What do you think about adding a test to check that the URL contains `lacework.net`?

Reply:
Done!